_posts/2015-07-23-postgres-review-operators.md

```diff
@@ -0,0 +1,292 @@
+---
+layout:     post
+title:      Review of operators in Postgres
+date:       2015-07-23 23:00:00
+summary:    In this post we will review some operators so we can have
```

Comment:
Fair point, this is the short description that is placed below the post title in the posts list.